webui/src/services/okrservice.js

```diff
@@ -216,7 +216,9 @@ export default class OkrService {
                 .then((body) => {
                     let sectionId = body.id;
                     this.setSubjectSectionId(subjectId, sectionId);
-                    this.shareNotebook(notebookId, errHandler);
+                    if (AUTO_SHARE_NOTEBOOK) {
+                        this.shareNotebook(notebookId, errHandler);
+                    }
```

Comment:
I don't remember, is it possible to check that notebook is already shared? Would be good to show message like 'Please share you notebook manually. To do that...."

Reply:
I guess it is possible to check sharing state using OneDrive API, but now I'm not sure that we need to keep it. `Read files` permission is still consented currently but it can make sense to remove in the future. It is need only to try to avoid "20k files error" of OneNote API.
So I think we need to wait little bit with the decision looking on people reaction.
I've added the task for future.